proxy/proxy/entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+echo "entrypoint.sh"
+
+sudo iptables -t nat -N proxy_redirect
+sudo iptables -t nat -A proxy_redirect -p tcp -d 104.18.29.94 -j REDIRECT --to-ports 8443
+sudo iptables -t nat -A proxy_redirect -p tcp -d 104.18.28.94 -j REDIRECT --to-ports 8443
+
+sudo iptables -t nat -N proxy_output
+sudo iptables -t nat -A proxy_output -p tcp --dport 6379 -j RETURN
+sudo iptables -t nat -A proxy_output -p tcp -m owner --gid-owner 123 -j RETURN
+sudo iptables -t nat -A proxy_output -j proxy_redirect
+
+sudo iptables -t nat -A OUTPUT -p tcp -j proxy_output
```

Comment:
Would it make sense to do this setup in an init container rather than in the entrypoint script? (thinking that might also remove the need for running the proxy as privileged...)

Reply:
Thanks!  It was in my to-do list.  I change it now.